nautobot_golden_config/models.py

```diff
@@ -138,7 +138,9 @@ def _verify_get_custom_compliance_data(compliance_details):
 
 def _get_hierconfig_remediation(obj):
     """Returns the remediating config."""
-    hierconfig_os = HIERCONFIG_LIB_MAPPER_REVERSE.get(obj.device.platform.slug)
+    hierconfig_os = HIERCONFIG_LIB_MAPPER_REVERSE.get(obj.device.platform.slug) or HIERCONFIG_LIB_MAPPER.get(
+        obj.device.platform.slug
+    )
```

Comment:
If you send in `ios` you don't want hierconfig to use `cisco_ios` as it needs the shorter version. You will probably want to do something like this instead:

```suggestion
    if obj.device.platform.slug in HIERCONFIG_LIB_MAPPER:
        hierconfig_os = obj.device.platform.slug
    else:
        hierconfig_os = HIERCONFIG_LIB_MAPPER_REVERSE.get(obj.device.platform.slug)
```

Reply:
You could also do `if obj.device.platform.slug in HIERCONFIG_LIB_MAPPER.values():` to save yourself the extra import.